src/components/Conf/Seo/index.tsx

```diff
@@ -41,6 +42,9 @@ export default function SeoConf(props: {
       <meta name="twitter:title" content={twitterTitle} />
       <meta name="twitter:description" content={description} />
       <meta name="twitter:image" content={image} />
+      {props.pathname && (
+        <link rel="canonical" href={`https://graphql.org/${props.pathname}`} />
```

Comment:
If `pathname` starts with a `/` then this will be an unpleasant URL. 
```suggestion
        <link rel="canonical" href={`https://graphql.org/${props.pathname.replace(/^[/]+/, '')}`} />
```

Reply:
previous solution didn't work, I replace by gatsby plugin 
<img width="668" alt="image" src="https://github.com/graphql/graphql.github.io/assets/7361780/fea688a6-2c67-4ae7-b912-95411f56e056">